eli5/explain.py

```diff
@@ -96,7 +96,9 @@ def explain_prediction(estimator, doc, **kwargs):
     doc : object
         Example to run estimator on. Estimator makes a prediction for this
         example, and :func:`explain_prediction` tries to show information
-        about this prediction.
+        about this prediction. Pass a single element, not a one-element array:
+        if you fitted your estimator on ``xs``, that would be ``xs[i]`` for
```

Comment:
scikit-learn docs use X instead of xs in most places; do you mean something different in ``xs``?